include/freerdp/settings.h

```diff
@@ -1214,7 +1221,11 @@ struct rdp_settings
 	ALIGN64 char* RedirectionAcceptedCert;        /* 1231 */
 	ALIGN64 UINT32 RedirectionAcceptedCertLength; /* 1232 */
 	ALIGN64 UINT32 RedirectionPreferType;         /* 1233 */
-	UINT64 padding1280[1280 - 1234];              /* 1234 */
+	ALIGN64 BYTE* RedirectionGuid;                /* 1234 */
```

Comment:
Could these fields be just after `RedirectionTsvUrl` to follow the same order as the documentation?
Down there, the `TargetNetAddressesCount` and `TargetNetAddresses` might make sense to add the `Redirection` prefix.
And I think `RedirectionAcceptedCert`, `RedirectionAcceptedCertLength` and  `RedirectionPreferType` are orphans fields that don't have relation with the Server Redirection PDU.